fabcar/javascript/uploadToDrive/fileUploadServer.js

```diff
@@ -0,0 +1,32 @@
+const express = require('express')
+const app = express()
+const bodyParser = require('body-parser')
+const cors = require('cors')
+const port = 3000
+const HOST = 'localhost'
+'use strict';
+
+const { Gateway, Wallets } = require('fabric-network');
+const path = require('path');
+const fs = require('fs');
+const FabricCAServices = require('fabric-ca-client');
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({ extended: true }));
+app.use(cors());
+
+var fileUpload = require('/home/shatabdi/Health-Blockchain/fabcar/javascript/uploadToDrive/encryptAndUpload.js')
+const method = fileUpload.method;
+
+app.post('/api/encryptAndUpload', function(req, res){
```

Comment:
And here you are not !

Reply:
this is correct, this is an express app and we are writing an api request. And calling functionality from a different file. Including everything in the express app will make it too bulky. I think its correct the way it is.